modules/costs/app/controllers/costlog_controller.rb

```diff
@@ -47,7 +47,7 @@ def create
     elsif @cost_entry.save
 
       flash[:notice] = t(:notice_cost_logged_successfully)
-      redirect_back fallback_location: work_package_path(@cost_entry.work_package)
+      redirect_to work_package_path(@cost_entry.work_package)
```

Comment:
Why that change? It does not look to be related.

Reply:
Because redirecting back results in `GET /projects/X/cost_entries/` which do not exist